### PR TITLE
Tiny graphical edit on in instrument tab

### PR DESCRIPTION
**Pull request by FredericGuilbault**
Instrument data touching the right side of the window feels broken. 
Like if some other text might be hidden. 
putting a space between last character and the edge of the window remove confusion.

before/after: 
https://imgur.com/a/WhLYb

### CODE REVIEW

**Comment by FredericGuilbault** [comment]
Oh, would you prefer PR in a separate branch ?

File: src/styles/app.less [hunk 1]
```diff
@@ -187,14 +187,14 @@ p {
     flex: 1 1 auto;
     font-size: 110%;
     color: @labelFontColor;
-    text-align: right;
+    text-align: center;
```
Comment (vokkim) [inline src/styles/app.less:190]: Setting `text-align: center` will make the text "jump" when text length/value changes. I'd still prefer aligning the text on right, but maybe adding some padding or margin to the right.

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "eslint-plugin-es5": "1.2.0",
     "eslint-plugin-react": "7.7.0",
     "express": "4.16.3",
+    "@geo-maps/countries-coastline-10km": "0.6.0",
```
Comment (vokkim) [inline package.json:52]: Is the `countries-coastline-10km` package necessary if the actual JSON file has been copied under `world-base.geo.json`? Maybe we could just add credits to the readme.

File: src/styles/app.less (hunk 1 above)
Comment (FredericGuilbault) [reply src/styles/app.less:190]: Fair point. 
See better approach on PR #14

File: package.json (hunk 2 above)
Comment (FredericGuilbault) [reply package.json:52]: I would go the other way. 
For more fluid maintenance in time. The JSON should be copied automatically on build or used strait from it's original location. 

Im not much used to nodejs but this seem the way to go. Although I woudn't know what would be the best strategy. So I did not push anything on this part exept adding the source package. 

It's an improvement but an incomplete feature. Maybe it should come with the opening of an issue.

Comment (FredericGuilbault) [reply package.json:52]: @vokkim What would be the best thing to do with this ? 

Im keeping it in my fork untill it's a full feature. 

You accept it like that and I open an issue about it on your repo.  

Or I remove the dependency, add credits to the readme and you will accept it.

**Pull request closed without merging**